scripts/performance_benchmark.py

```
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+"""
+Performance benchmark script for large dataset analysis.
+
+This script measures the actual performance characteristics of the TableConverter
+with various data sizes to understand the current limitations and guide the
+implementation of performance safeguards.
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)